build.sh

```diff
@@ -23,25 +21,30 @@ install_dir=/tmp/scitt
 mkdir -p $install_dir
 
 mkdir -p build/attested-fetch
-CC="$CC" CXX="$CXX" \
-    cmake -GNinja -B build/attested-fetch \
-    -DCMAKE_BUILD_TYPE="${CMAKE_BUILD_TYPE}" \
-    -DCMAKE_INSTALL_PREFIX=$install_dir \
-    -DCOMPILE_TARGET="${PLATFORM}" \
-    "$root_dir/3rdparty/attested-fetch"
-
-ninja -C build/attested-fetch ${NINJA_FLAGS} --verbose
-ninja -C build/attested-fetch ${NINJA_FLAGS} install
 
 if [ "$PLATFORM" = "sgx" ]; then
     ATTESTED_FETCH_MRENCLAVE_HEX=$(/opt/openenclave/bin/oesign dump -e $install_dir/libafetch.enclave.so.signed | sed -n "s/mrenclave=//p")
+    CC=${CC:-clang-11}
+    CXX=${CXX:-clang++-11}
 elif [ "$PLATFORM" = "virtual" ]; then
     ATTESTED_FETCH_MRENCLAVE_HEX=""
+    CC=${CC:-clang-15}
```

Comment:
what is the reasoning for this change? why make it different from SGX?

Reply:
It's a requirement of CCF 3.0.13: [Releases · microsoft/CCF (github.com)](https://github.com/microsoft/CCF/releases)

 clang-11 for SGX and clang-15 for virtual